modules/installation-registry-storage-block-recreate-rollout.adoc

```diff
@@ -32,5 +32,45 @@ $ oc patch config.imageregistry.operator.openshift.io/cluster --type=merge -p '{
 ----
 +
 . Provision the PV for the block storage device, and create a PVC for that volume. The requested block volume uses the ReadWriteOnce (RWO) access mode.
+.. You can define a VMware vSphere PersistentVolumeClaim by creating a file, `pvc.yaml`, with the following contents:
 +
-. Edit the registry configuration so that it references the correct PVC.
+[source,yaml]
+----
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: image-registry-storage <1>
+spec:
+  accessModes:
+  - ReadWriteOnce <2>
+  resources:
+    requests:
+      storage: 100Gi <3>
+----
+<1> A unique name that represents the PersistentVolumeClaim.
+<2> The PersistentVolumeClaim's access mode. With `ReadWriteOnce`, the volume can be mounted with read and write permissions by a single node.
```

Comment:
Suggestion to remove the possessive on the PVC:

s/The PersistentVolumeClaim's access mode./The access mode of the PersistentVolumeClaim./

Also is parallel in structure to <3>.